backend/routes/model.js

```diff
@@ -379,6 +380,91 @@ router.get("/:model/:uid.json.mpc",  middlewares.hasReadAccessToModel, getJsonMp
 
 router.get("/:model/:uid.unity3d", middlewares.hasReadAccessToModel, getUnityBundle);
 
+router.get("/:model/:uid.gltf", middlewares.hasReadAccessToModel, getGLTF);
+router.get("/:model/:uid.bin", middlewares.hasReadAccessToModel, getGLTFBin);
+
+router.get("/:model/:uid.src.mpc", middlewares.hasReadAccessToModel, getSRC);
+
+// Shape Resource Container information
+
+/**
+ * @api {get} /:teamspace/:model/revision/:rev/srcAssets.json Get revision's src assets
+ * @apiName getRevSrcAssets
+ * @apiGroup Model
+ * @apiDescription Get the model's assets but of a particular revision
+ *
+ * @apiParam {String} teamspace Name of teamspace
+ * @apiParam {String} model The model Id to get unity assets for.
+ * @apiParam {String} rev The revision of the model to get src assets for
+ *
+ * @apiExample {get} Example usage:
+ * GET /teamSpace1/3549ddf6-885d-4977-87f1-eeac43a0e818/revision/07314a08-75da-46bf-9d2c-b7b20109143b/srcAssets.json HTTP/1.1
+ *
+ * @apiSuccessExample {json} Success:
+ * {
+ *   "models": [
```

Comment:
@sebjf I got a couple questions:

1. How are you getting the model offset at the moment? The unityAsset.json version has the offset listed in here, from this example it doesn't look like you're extracting that? I looked into the Unreal code and it wasn't immedately clear where it's set.
2. Can we change the schema of this return? ideally this should be more inline with unityAssets, atm there's a lot of redundant info (i'll change it on the web side, just making sure you're happy with it on the unreal code change):
````json
{ "models": [
    {
         "database" : "sebjf",
          "model" : "9b458610-0c94-11eb-a083-2b878a4dc41e",
          "srcIDs" : [ "153cf665-2c84-4ff9-a9e2-ba495af8e6dc", "2967230f-67fa-45dc-9686-161e45c7c8a2", "07c67b6c-4b02-435f-8639-ea88403c36f7"]
    }
]}
````